### ISSUE TITLE
AppImage update dialog links me to .deb - maybe that would be worth changing?

### ISSUE THREAD

**Issue by ell1e** [blizzard4591/openMittsu#67]
The AppImage update dialog links me to a .deb download, which I cannot use on this rpm distro. Maybe that would be worth changing to actually link me to the AppImage version again? (On top of that, I also did download the AppImage just minutes ago from the link in the README, so there might be some potential issue with that not being updated. Noting it just in case that needs fixing too)

**Comment by blizzard4591** [comment]
Yes, the AppImage and update is somewhat of a mystery to me. The binary itself currently has no way of knowing whether it is a standard debian-build or inside an AppImage, maybe I need a distinct flag for AppImage. Will look into.

**Comment by poVoq** [comment]
Appimages are not available any longer? Would be nice to have for non-debian distributons.
